### PR TITLE
Jquery: Add article aliases(redirects) for append,after,before (#610)

### PR DESCRIPTION
**Pull request by pvskarthikeya**
## Related Issues and Discussions
[#610](https://github.com/duckduckgo/zeroclickinfo-fathead/issues/610)

## People to notify
@bfmags 
@MariagraziaAlastra 

## Testing & Review
To be completed by Language Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page**
- [ ] Instant Answer page is correctly filled out and contains:
    - One topic for the Search Space Language (Java, Python, Scala, Ruby, etc.)
    - One topic from: Reference, Help, Libraries, Tools
        - Documentation Fatheads are considered "Reference"
    - Description, Data source, and 2+ example queries
    - Perl Module (e.g. "DDG::Fathead::PerlDoc" -- we only need a name, not an actual file)
    - Source Name (for "More at <source_name>" link)
    - Source Domain (must contain http:// or https:// -- can be the same as Data Source)
    - Source Info (used as Subtitle for each Article -- usually matches the IA Name)
    - 'Skip Abstract' is checked off
    - Source ID (ping @moollaza to assign one, once Fathead is ready for Beta deploy)
- [ ] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Code**
- [x] Uniformly indented, well commented
- [x] Fetch.sh and Parse.sh run without errors
- [x] Output contains no blank lines, or multi-line entries

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/jquery

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pvskarthikeya Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/fathead/jquery/redirects.txt [hunk 1]
```diff
@@ -298,10 +307,32 @@ and self example,.andSelf()
...
+append elements,.append()
+append to html,.append()
+append to html element,.append()
+append to div,.append()
+append to page,.append()
+append to dom,.append()
+insert as last child,.append()
+insert in to element,.append()
+insert in to DOM element,.append()
+insert in to html element,.append()
```
Comment (moollaza) [inline lib/fathead/jquery/redirects.txt]: please use `into` as well:

```
insert in to html element,.append()
insert into html element,.append()
...
```

File: lib/fathead/jquery/redirects.txt [hunk 2]
```diff
@@ -298,10 +307,32 @@ and self example,.andSelf()
...
 append to example,.appendTo()
 appendTo example,.appendTo()
 attr example,.attr()
 before example,.before()
+insert before element,.after()
+insert before html element,.after()
+insert before div,.after()
+insert before dom element,.after()
+add element before,.after()
```
Comment (moollaza) [inline lib/fathead/jquery/redirects.txt]: @pvskarthikeya looks like this should be `, .before()` instead of `.after()`?

**Comment by pvskarthikeya** [comment]
Hey! @moollaza. I will make the changes now. Thanks.

**Comment by moollaza** [comment]
@pvskarthikeya Thanks! This LGTM :+1:

Nicely done 💯

**Comment by pvskarthikeya** [comment]
@moollaza Thank you. 😃

**Pull request merged**